<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtArteryTechPreconditionerSetupUI</class>
 <widget class="QDialog" name="QtArteryTechPreconditionerSetupUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>255</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtArteryTechPreconditionerSetupUI</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="PreconditionerTypeLabel">
           <property name="text">
            <string>Preconditioner Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="PreconditionerTypeLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="ASMTypeLabel">
           <property name="text">
            <string>ASM Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ASMTypeLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="SubdomainSolverLabel">
           <property name="text">
            <string>Subdomain Solver:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="SubdomainSolverLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="ILULevelsLabel">
           <property name="text">
            <string>ILU Levels:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="ILULevelsLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="SubdomainMatrixOrderingTypeLabel">
           <property name="text">
            <string>Subdomain Matrix Ordering Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="SubdomainMatrixOrderingTypeLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="OverlappingTypeLabel">
           <property name="text">
            <string>Overlapping Type:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="OverlappingTypeLineEdit"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QLabel" name="OverlappingSizeLabel">
           <property name="text">
            <string>Overlapping Size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="OverlappingSizeLineEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="OKPushButton">
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="CancelPushButton">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
